### ISSUE TITLE
can't download message 

### ISSUE THREAD

**Issue by bodarycomforts** [proshunsuke/colmsg#54]
C:\Users\...\colmsg.exe -n 大園玲 --dir G:\colmsg --h_refresh_token <h_refresh_token> --s_refresh_token <s_refresh_token>

←31m[colmsg error]←[0: error: missing field `member_id` at line 1 column 1551, request url:

**Comment by proshunsuke** [comment]
thank you for error report.
I also faced the same problem.

**Comment by proshunsuke** [comment]
@bodarycomforts 
I released [v2.0.4](https://github.com/proshunsuke/colmsg/releases/tag/v2.0.4)

```
> colmsg -g sakurazaka -n 大園玲
saving messages of 大園玲...
complete saving messages of 大園玲!
```

**Comment by bodarycomforts** [comment]
arigatou~
